user-manager/src/main/java/com/qpay/usermanager/model/dto/merchant/MerchantCreation.java

```diff
@@ -0,0 +1,18 @@
+package com.qpay.usermanager.model.dto.merchant;
+
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.*;
+
+@Data
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor(access = AccessLevel.PROTECTED)
+@Builder
+public final class MerchantCreation {
+
+    private @NotBlank(message = "Name is mandatory") String name;
+
+    private @NotBlank(message = "Email is mandatory") String email;
+
+    private @NotBlank(message = "Password is mandatory") String password;
+}
```

Comment:
`MerchantCreation` should be `record`. look at `CustomerModification`

Reply:
andrei said me to use builder